assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -8,7 +8,7 @@
  * Controller of the openshiftConsole
  */
 angular.module('openshiftConsole')
-  .controller('NewFromTemplateController', function ($scope, $http, $routeParams, DataService, $q, $location, TaskList, $parse, Navigate) {
+  .controller('NewFromTemplateController', function ($scope, $http, $routeParams, DataService, $q, $location, TaskList, $parse, Navigate, $filter) {
```

Comment:
look up the `displayName` filter once and save it in a var:
`var displayNameFilter = $filter('displayName');`